dpbench/benchmarks/dbscan/dbscan_numba_dpex_n.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 Intel Corporation
```

Comment:
For dbscan we only have kernel and prange implementations; no numpy implementation. dbscan_numba_dpex_n.py is the same as dbscan_numba_dpex_p.py. So we can remove dbscan_numba_dpex_n.py

Reply:
Removed file.